src/ck3/data/courtier_guest_mgmt.rs

```diff
@@ -0,0 +1,88 @@
+use crate::block::Block;
+use crate::context::ScopeContext;
+use crate::db::{Db, DbKind};
+use crate::desc::validate_desc;
+use crate::everything::Everything;
+use crate::game::GameFlags;
+use crate::item::{Item, ItemLoader};
+use crate::report::{err, ErrorKey};
+use crate::scopes::Scopes;
+use crate::token::Token;
+use crate::tooltipped::Tooltipped;
+use crate::trigger::validate_trigger;
+use crate::validate::validate_modifiers_with_base;
+use crate::validator::Validator;
+
+#[derive(Clone, Debug)]
+pub struct CourtierGuestManagement {}
+
+inventory::submit! {
+    ItemLoader::Normal(GameFlags::Ck3, Item::CourtierGuestManagement, CourtierGuestManagement::add)
+}
+
+impl CourtierGuestManagement {
+    pub fn add(db: &mut Db, key: Token, block: Block) {
+        db.add(Item::NameList, key, block, Box::new(Self {}));
```

Comment:
Shouldn't this be `Item::CourtierGuestManagement`?

Reply:
Oops yes